backend/app/routers/cameras.py

```
@@ -151,6 +174,94 @@ def delete_camera(camera_id: int) -> Dict[str, Any]:
         raise HTTPException(status_code=500, detail=f"Failed to delete camera: {e}") from e
 
 
+# ---------- Zones (ROIs) for crowd management ----------
+
+@router.get("/{camera_id}/zones", response_model=List[Dict[str, Any]])
+def list_zones(camera_id: int) -> List[Dict[str, Any]]:
+    """List all zones (ROIs) for a camera."""
+    try:
+        docs = list(zones_col.find({"camera_id": camera_id}, {"_id": 0}))
+        return docs
```

Ruff (0.15.1)

[warning] Consider moving this statement to an `else` block

(TRY300)